src/main/java/io/dropwizard/kafka/KafkaConsumerBundle.java

```diff
@@ -44,7 +45,7 @@ public void initialize(final Bootstrap<?> bootstrap) {
     }
 
     @Override
-    public void run(final T configuration, final Environment environment) throws Exception {
```

Comment:
`throws Exception` is declared but never thrown in the method. Code smell.

Reply:
The method signature is defined as part of the `ConfiguredBundle` interface.